src/main/lua/exasolvs/query_renderer.lua

```diff
@@ -57,15 +57,15 @@ end
 -- @return new query renderer instance
 --
 function M.new (query)
-    local self = {original_query = query, query_elements = {}}
+    local self = { original_query = query, query_elements = {} }
```

Comment:
```suggestion
    local self = {original_query = query, query_elements = {}}
```

Please keep the style uniform.